osmnx/bearing.py

```diff
@@ -281,6 +282,6 @@ def _bearings_distribution(
     count = np.roll(count, 1)
     bin_counts = count[::2] + count[1::2]
 
-    # because we merged the bins, their edges are now only every other one
-    bin_edges = bin_edges[range(0, len(bin_edges), 2)]
-    return bin_counts, bin_edges
+    # because we merged the bins, their centers are now only every other one
+    bin_centers = bin_edges[range(0, len(bin_edges) - 1, 2)]
```

Comment:
`len(bin_edges)` is always even due to the double-bin implementation. It is 2 * num_bins.

If user inputs num_bins=1, then `range(0, len(bin_edges) - 1, 2)` return `0` (there is only a single bin centered at 0 degrees.

If user inputs num_bins=2, then `range(0, len(bin_edges) - 1, 2)` returns `0, 180`

Reply:
Ok I think that makes sense now.